lib/kafo_parsers/puppet_strings_module_parser.rb

```diff
@@ -42,6 +42,7 @@ def initialize(file)
         raise KafoParsers::ParseError, "'#{command}' returned error\n#{@raw_json}"
       end
 
+      Encoding.default_external = Encoding::UTF_8 if defined?(Encoding) && Encoding.respond_to?(:default_external=)
```

Comment:
on what Ruby version `default_external=` is not defined? is that defensive check?